packages/next/build/webpack/plugins/terser-webpack-plugin/src/worker.js

```diff
@@ -1,21 +1,2 @@
 import minify from './minify'
-
-module.exports = (options, callback) => {
-  try {
-    // 'use strict' => this === undefined (Clean Scope)
-    // Safer for possible security issues, albeit not critical at all here
-    // eslint-disable-next-line no-new-func, no-param-reassign
-    options = new Function(
-      'exports',
-      'require',
-      'module',
-      '__filename',
-      '__dirname',
-      `'use strict'\nreturn ${options}`
-    )(exports, require, module, __filename, __dirname)
-
-    callback(null, minify(options))
-  } catch (errors) {
-    callback(errors)
-  }
-}
+export default options => minify(options)
```

Comment:
Can we unwind this shell file?

Reply:
What do you mean by unwind?

Reply:
Just delete it and make `jest-worker` call the `minify.js` file directly.

Reply:
Ah yeah makes sense 👍